### PR TITLE
Add tests in server

### PR DESCRIPTION
**Pull request by AlexGeb**

### CODE REVIEW

File: packages/server/src/controllers/accounts.ts [hunk 1]
```diff
@@ -5,23 +5,18 @@ import { extractSessionId } from "../middlewares/authenticate";
...
-      account: {
-        firstName,
-        lastName,
-      },
-    };
-    ctx.status = 201;
-  } catch (error) {
-    ctx.status = 500;
-    ctx.message = "Error when creating account";
-  }
```
Comment (paradoux) [inline packages/server/src/controllers/accounts.ts:24]: Why removing the error handling ? :))

File: packages/server/src/validation.ts [hunk 2]
```diff
@@ -10,4 +10,4 @@ export const accountSchema = Joi.object({
     })
     .required(),
   password: Joi.string().alphanum().min(6).max(30).required(),
-});
+}).required();
```
Comment (paradoux) [inline packages/server/src/validation.ts:13]: What does this required() do ? :))

File: packages/server/src/controllers/accounts.ts (hunk 1 above)
Comment (AlexGeb) [reply packages/server/src/controllers/accounts.ts:24]: It was simpler for debugging. I suggest to never catch errors and only handle them at the end. It is done in index.ts: 
app.on("error", (err) => {
  console.log(err);

File: packages/server/src/validation.ts (hunk 2 above)
Comment (AlexGeb) [reply packages/server/src/validation.ts:13]: An empty body would pass this validation if we don't require it!

File: packages/server/src/controllers/accounts.ts (hunk 1 above)
Comment (paradoux) [reply packages/server/src/controllers/accounts.ts:24]: Mmmh not sure it's that great for logging/monitoring purposes + security to just log the error in the index level.
What I can do is leave all the error catching in the handler level but add a log of the error object with serialized stack trace for debugging purposes :))

File: packages/server/src/validation.ts (hunk 2 above)
Comment (paradoux) [reply packages/server/src/validation.ts:13]: Ooh I see good one ! 👍

File: packages/server/src/controllers/accounts.ts (hunk 1 above)
Comment (AlexGeb) [reply packages/server/src/controllers/accounts.ts]: Indeed, we need a proper way of handling errors. 
I have put back the try-catch.

**Pull request merged**